static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,144 @@
+shared cache storage on the NFS server that will allow your team to share and
+store data for your projects effectively as possible and have a workspace
+restoration/switching speed as instant as `git checkout` for your code.
+
+With large data files it is better to set the cache directory to external NFS.
+Not only just it will cache the data faster but also version the data. Suppose,
+we have a dataset with 1 million images. With DVC, we can have multiple versions
+of a dataset without affecting each other work and without creating duplicates
+of a complete dataset. With `cache directory` set to `NFS server` you would
+avoid copying large files from NFS server to the machine and DVC will manage the
+links from the workspace to cache. For more information, visit
+[Data and Model Files Versioning](/doc/use-cases/data-and-model-files-versioning).
+
+## Preparation
+
+First configure NFS server and client machine, following this
+[link](https://vitux.com/install-nfs-server-and-client-on-ubuntu/).
+
+In order to make it work on a shared server, after configuring NFS server and
+client we need to setup a shared cache location for your projects, so that every
+team member is using the same cache location.
+
+After configuring NFS on both server and client side. Let's create an export
+directory on server side where all data will be stored.
+
+```dvc
+$ mkdir -p /storage
+```
+
+You will have to make sure that the directory has proper permissions setup,so
+that every one on your team can read and write to it and can access cache files
+written by others. The most straightforward way to do that is to make sure that
+you and your colleagues are members of the same group (e.g. 'users') and that
+your shared directory is owned by that group and has respective permissions.
+
+Let's create a mount point of client side.
+
+```dvc
+$ mkdir -p /mnt/dataset/
+```
+
+From `/mnt/dataset/` you will be able to access `/storage` directory present in
+host server from your local machine.
+
+## Configuring Cache location
+
+After mounting the shared directory on client side. Assuming project code is
+present in `/home/user/project1`. Let's initialize a `dvc repo`.
+
+```dvc
+$ cd /home/user/project1/
+$ git init
+$ dvc init
+$ git add .dvc .gitignore
+$ git commit . -m "initialize DVC"
+```
+
+With `dvc init`, we initialized a DVC repository. DVC will start tracking all
+the changes.
+
+Tell DVC to use the directory we've set up as an external cache location by
+running:
+
+```dvc
+$ dvc config cache.dir /mnt/dataset/storage
+$ dvc config cache.type "reflink,symlink,hardlink,copy"
+$ dvc config cache.protected true
+$ git add .dvc .gitignore
+$ git commit . -m "DVC cache location updated"
+```
+
+By default cache is present in the `.dvc/cache` location. `dvc cache dir`
+changes the location of cache directory to `/mnt/dataset/storage`
+
+`config cache.dir /path/to/cache/directory` - sets cache directory location.
```

Comment:
no need to complicate it here with alternatives, let's use `dvc cache dir`

Reply:
done